test/models/profile_test.rb

```diff
@@ -18,15 +18,6 @@ class ProfileTest < ActiveSupport::TestCase
     assert_not profiles(:one).compliant?(hosts(:one))
   end
 
-  test 'host is compliant if all rules are "pass" or "notapplicable"' do
-    test_results(:one).update(host: hosts(:one), profile: profiles(:one))
-    RuleResult.create(rule: rules(:one), host: hosts(:one), result: 'pass',
-                      test_result: test_results(:one))
-    RuleResult.create(rule: rules(:two), host: hosts(:one),
-                      test_result: test_results(:one), result: 'notapplicable')
-    assert profiles(:one).compliant?(hosts(:one))
-  end
```

Comment:
This test no longer makes sense, as compliance is based solely on the TestResult score